ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -177,6 +211,15 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                                                 services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'tree'}});
                                                             }}
                                                         />
+                                                        <i
+                                                            className={classNames('fa fa-object-group', {selected: pref.view === 'compact'})}
+                                                            title='Compact'
+                                                            onClick={() => {
+                                                                clearFilter();
```

Comment:
Is it necessary to clear the Filter here? If the user inadvertently selects this view, the user will lose the filter they set. Moving between tree to pod to network to list views does not clear the filter.